kafka-rest/src/main/java/io/confluent/kafkarest/DefaultKafkaRestContext.java

```diff
@@ -47,7 +47,7 @@ public class DefaultKafkaRestContext implements KafkaRestContext {
   private SchemaRegistryClient schemaRegistryClient;
 
   public DefaultKafkaRestContext(KafkaRestConfig config) {
-    log.debug("Creating context with config: {}", config);
```

Comment:
this doesn't actually log the configuration at the moment due to it use default toString() of Object, but changing this just in case.